### PR TITLE
Scroll towards the current action in the dropdown

### PR DESCRIPTION
**Pull request by saraycp**
Especially in dropdowns with a lot of request actions, it's helpful to have the dropdown centred on the current action (the one we are currently displaying on the page).

scroll_to_active_request_action

### CODE REVIEW

File: src/api/app/views/webui/request/_actions_details.html.haml
Comment (saraycp) [inline src/api/app/views/webui/request/_actions_details.html.haml:22]: This change is also introduced in https://github.com/openSUSE/open-build-service/pull/13681.
Let's keep an eye on the conflicts.

Comment (dmarcoux) [reply src/api/app/views/webui/request/_actions_details.html.haml:22]: There's an extra change in https://github.com/openSUSE/open-build-service/pull/13681/commits/606293751399a8a9910e919eb0c41dc7c3867bf0 with `.border-top` on the `<li>` element. The commit message/description are also more descriptive of what was done and why it was done, instead of how the changes were done.

**Comment by saraycp** [comment]
DO NOT MERGE until #13681 is merged. This PR is based on it.

**Comment by dmarcoux** [comment]
This PR can be rebased now.

**Comment by saraycp** [comment]
Rebased and ready for reviews now.

File: src/api/app/assets/javascripts/webui/request_show_redesign/action_selector.js
Comment (dmarcoux) [inline src/api/app/assets/javascripts/webui/request_show_redesign/action_selector.js:2]: This works great :+1:.

Could we move this code to the view/component where it is needed? Otherwise, it's processed on every page, although nothing obviously happens since `$('.dropdown#request-actions')` doesn't match.

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**
